docs/installation_guide.rst

```diff
@@ -472,7 +472,7 @@ then do::
     Make sure that pysqlcipher3 is properly installed. If ``$ pip3 freeze | grep pysqlcipher3`` returns nothing for you then it was not installed. Try to manually install only that dependency with the verbose option to see where it fails. ``$ pip3 install pysqlcipher3 -v``. If it fails at the stage of finding the library for ``-lsqlcipher`` or ``sqlcipher/sqlite3.h file not found`` then ``brew install sqlciper`` did not place the installed lib directory to the ``LIBRARY_PATH`` and you will have to do it manually.
     For example, if ``sqlcipher`` was installed using Homebrew, the library path resembles this ``/opt/homebrew/Cellar/sqlcipher/<version>/lib`` on Mac ARM based chips and ``/usr/local/Cellar/sqlcipher/<version>/lib`` on Mac Intel based chips by default.
     Afterwards, install ``pysqlcipher3`` using the command below:
-    ``$ LIBRARY_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/lib pip3 install pysqlcipher3``.
+    ``$ LIBRARY_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/lib C_INCLUDE_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/include pip3 install pysqlcipher3``.
```

Comment:
what is the `C_INCLUDE_PATH` needed for? We have not had the need to use it (afaik) in our macos CI building. @kelsos ?

Reply:
Not adding the `C_INCLUDE_PATH` did not install `pysqlcipher3` for me. 
>In file included from src/python3/connection.c:27:
    src/python3/connection.h:33:10: fatal error: 'sqlcipher/sqlite3.h' file not found
    #include "sqlcipher/sqlite3.h"
             ^~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit code 1
    Running setup.py install for pysqlcipher3 ... done
Successfully installed pysqlcipher3
WARNING: You are using pip version 21.2.4; however, version 22.1.1 is available.
You should consider upgrading via the '/Users/Isaac/Desktop/rotki/.venv/bin/python -m pip install --upgrade pip' command.
❯ python -c "import pysqlcipher3"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pysqlcipher3'